pyproject.toml

```diff
@@ -196,6 +202,10 @@ ignore = [  # NOTE: non-permeanent exclusions should be added to `.ruff.toml` in
     "T203"    # pprint found
 ]
 
+[tool.ruff.flake8-annotations]
```

Comment:
Please also update the flake8 config file

Reply:
@eerovaher. Can you address this comment. Then I can do what looks to be a final review / approval. Thanks!

Reply:
The only reason I would be willing to touch our Flake8 configuration would be to delete it entirely. We have been using Ruff for some time now, I see no value in maintaining configuration for another Python linter.

Reply:
I agree but I think should be dealt with in another PR where the advocates for keeping it can provide input. In the meantime shouldn't we keep it synced?

Reply:
It's clearly not being kept in sync with the Ruff configuration by anyone.

Reply:
@pllim. Alright to merge ?